bodhi/tests/test_masher.py

```diff
@@ -1009,7 +1028,7 @@ def test_push_timestamps(self, publish, *args):
         publish.assert_called_with(
             topic="mashtask.complete",
             force=True,
-            msg=dict(success=True, repo='f17-updates-testing'))
+            msg=dict(success=True, repo='f17-updates-testing', agent='lmacken'))
```

Comment:
This kinda surprises me a little since the MasherThread above seems to be created by ``ralph``. Is this expected?